SwiftyStoreKit/SwiftyStoreKit.swift

```diff
@@ -158,7 +158,26 @@ public class SwiftyStoreKit {
         password: String? = nil,
         session: NSURLSession = NSURLSession.sharedSession(),
         completion:(result: VerifyReceiptResult) -> ()) {
-            InAppReceipt.verify(receiptVerifyURL: url, password: password, session: session, completion: completion)
+        InAppReceipt.verify(receiptVerifyURL: url, password: password, session: session) { result in
+         
+            dispatch_async(dispatch_get_main_queue()) {
+                completion(result: result)
```

Comment:
This change is technically not part of this PR but I realised it's a fix that is needed otherwise we the completion block is not called on the main thread.